packages/graphql-serve/.graphqlrc.yml

```diff
@@ -0,0 +1,22 @@
+schema: ./server/src/schema/*.graphql
```

Comment:
Why this file here?

Reply:
See comment bellow. We do not need it here. We can use json for defaults

Reply:
This is still pushed?

Reply:
I will remove it in the next push. It is not needed anymore